### PR TITLE
FIX warning in mass editing module when read

### PR DESCRIPTION
**Pull request by bealdav**
Just a small fix to remove Warning.

It also avoid Sentry users to send false positive alerts.

sentry

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

**Comment by bealdav** [comment]
the same fix for v8 https://github.com/OCA/server-tools/pull/739.
unit tests are ok, really small decrease of codecov

**Comment by bealdav** [comment]
ping @JayVora-SerpentCS could you make a review, please ?

File: mass_editing/wizard/mass_editing_wizard.py [hunk 1]
```diff
@@ -256,3 +256,14 @@ def create(self, vals):
...
+
+    def read(self, fields, load='_classic_read'):
+        """ Without this call, dynamic fields build by fields_view_get()
+            generate a log warning, i.e.:
+            odoo.models:mass.editing.wizard.read() with unknown field 'myfield'
+            odoo.models:mass.editing.wizard.read()
+                with unknown field 'selection__myfield'
+        """
+        # We remove fields which are not in _fields
+        real_fields = [x for x in fields if x in self._fields]
```
Comment (StefanRijnhart) [inline mass_editing/wizard/mass_editing_wizard.py]: 'fields' could be None. Can you cover this case?

**Comment by bealdav** [comment]
you can squash, when merge, thanks

**Comment by JayVora-SerpentCS** [comment]
LGTM

**Comment by pedrobaeza** [comment]
Someone that understands me, hehe!!

**Review by StefanRijnhart — APPROVED** [review]

**Pull request merged**